ground/openpilotgcs/src/libs/libs.pro

```diff
@@ -13,4 +13,6 @@ SUBDIRS   = \
     qwt \
     sdlgamepad
 
-SUBDIRS +=
+exists( $(OSG_SDK_DIR) ) {
```

Comment:
because that environment variable won't be defined because it doesn't use top level makefile